src/javascript/JContent/ContentRoute/ContentLayout/ContentLayout.container.jsx

```diff
@@ -306,7 +307,7 @@ ContentLayoutContainer.propTypes = {
     sort: PropTypes.object.isRequired,
     t: PropTypes.func.isRequired,
     uilang: PropTypes.string.isRequired,
-    treeState: PropTypes.number.isRequired,
+    treeState: PropTypes.number,
```

Comment:
Why did you remove the isRequired?

Reply:
proptype error in console that anoyed me. As the component work well without this props, I removed the required here.

Reply:
Ok could you add a default value then :)

Reply:
What kind of default value ?

Reply:
The property has been removed actually